src/main/java/core/basesyntax/service/file/FileReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.file;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public String readFromFile(String path) {
+        StringBuilder stringBuilder = new StringBuilder();
+        boolean firstLine = true;
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(path))) {
+            String fruitInfo = bufferedReader.readLine();
+
+            while (fruitInfo != null) {
+                if (firstLine) {
+                    firstLine = false;
+                    fruitInfo = bufferedReader.readLine();
+                    continue;
+                }
```

Comment:
remove this logic of skipping the first line 
you can do this simpler by calling readline() outside of the loop, but a better idea would be to skip the first line of the list in the service that's responsible for parsing transactions, and let this method just read the whole file as is